### ISSUE TITLE
vrcft module spits out completely arbitrary values not aligned with there name, or use case

### ISSUE THREAD

**Issue by hoshimi-kiyoshi** [regzo2/VRCFaceTracking-MeowFace#12]
just finally got this working, and found that almost all parameters being sent to vrc are all scrambled-
eye tracking works, but that's just about it
the eyebrows move the jaw to the right
sticking tongue out closes left eye
countless things close the right eye
squinting lowers the eyebrows... 

____etc you get the point
(and yes ive checked, the avatar is set up correctly, same result on other avis too)

https://github.com/user-attachments/assets/1429a763-144b-409d-89ad-8ea6ef028a9d

i dont know enough about c# to debug anything myself

<3~ thanks if this is ever updated

**Comment by hoshimi-kiyoshi** [comment]
only update is I attempted to build my own version and see if that runs properly, but it kept throwing out errors it couldn't find a dependency version that doesn't exist, I believe it was something like Jason.text 4.0.0.0 was a few months ago so I don't remember.

tldr tryed to fix, never got anything working, sorry-

I just ended up using unity face tracking on an old iPhone se I had lieing around, good luck
